google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -83,10 +94,11 @@ def get_all_violations(self, timestamp, resource_name):
         """Get all the violations.
 
         Args:
-            timestamp: The timestamp of the snapshot.
+            timestamp (str): The timestamp of the snapshot.
+            resource_name (str): String that defines a resource.
 
         Returns:
-             A tuple of the violations as dict.
+            tuple: A tuple of the violations as dict.
```

Comment:
For some reason, nenad tried this and lint is complaining.  So, sticking with what's there.